integration-tests/src/ethereum/ctc/contracts/GluwaCreditVestingToken.sol/CreditcoinBase.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Does this work without the "dbg" files? If so, lets remove them.

Reply:
Yes, it works. Removed.